ttbarAC_skim/plugins/EventSaverFlatNtuple.cxx

```diff
@@ -373,9 +374,11 @@ void EventSaverFlatNtuple::analyze( const edm::Event& event, const edm::EventSet
     }
 
     // Missing Transverse Momentum
-    m_met_met = (*m_met.product())[0].pt();
-    m_met_phi = (*m_met.product())[0].phi();
-
+    const pat::MET &met = m_met->front();
```

Comment:
Ultimately this did not seem necessary, but we'll keep it this way since the MET selector wasn't doing anything.